src/System.Transactions.Local/src/System/Transactions/EnlistmentState.cs

```diff
@@ -32,25 +32,8 @@ private static object ClassSyncObject
             }
         }
 
```

Comment:
Seems like you could remove this property since s_classSyncObject will be initialized by LazyInitializer.EnsureInitialized if it's null.